Soccar [UI]/src/nl/socnet/message/handler/StartGameMessageHandler.java

```diff
@@ -0,0 +1,28 @@
+package nl.socnet.message.handler;
+
+import io.netty.buffer.ByteBuf;
+import nl.soccar.socnet.connection.Connection;
+import nl.soccar.socnet.message.MessageHandler;
+import nl.socnet.message.StartGameMessage;
+
+/**
+ *
+ * @author PTS34A
+ */
+public final class StartGameMessageHandler extends MessageHandler<StartGameMessage> {
+
+    @Override
+    protected void handle(Connection connection, StartGameMessage message) throws Exception {
+        throw new UnsupportedOperationException("Handling is not supported for the Client.");
+    }
+
+    @Override
+    protected void encode(Connection connection, StartGameMessage message, ByteBuf buf) throws Exception {

```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Add a nested comment explaining why this method is empty, throw an UnsupportedOperationException or complete the implementation. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://194.135.91.67:9000/coding_rules#rule_key=squid%3AS1186)